live-examples/css-examples/css/word-break.css

```diff
@@ -0,0 +1,5 @@
+#example-element {
+ width:10em;
+ text-align: start;
+ border: solid 1px darkgray;
```

Comment:
Nit: 4 spaces of indentation in CSS

Reply:
Yes, I don't know where those "disappeared". Kinda sorry about that.